be/src/storage/rowset/segment_v2/column_writer.cpp

```diff
@@ -546,7 +568,7 @@ Status ScalarColumnWriter::finish_current_page() {
         size_t num_null = data_page_footer->num_values() - num_data;
         // If more than 80% of the current page is NULL records, using format 1 for the next page,
         // otherwise using format 2.
-        _curr_page_format = (num_null > 4 * num_data) ? 1 : 2;
+        _curr_page_format = (num_null > 4 * num_data) ? 1 : config::default_page_format;
```

Comment:
If one be is page_format==3, another be is page_format == 2, how to handle the clone logic?